common/src/main/java/org/dash/wallet/common/data/TaxCategory.kt

```diff
@@ -43,6 +43,10 @@ enum class TaxCategory(val value: Int) {
             return values()[value];
         }
 
+        fun fromValue(name: String): TaxCategory? {
+            return values().find { it.name.lowercase() == name.lowercase() }
+        }
```

Comment:
This allows TaxCategories to be stored as strings on Platform.  

We may want to change to numbers, which could save some fees.